docs/src/modules/utils/useCustomizationPlayground.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import pick from 'lodash/pick';
+
+export type CustomizationOptions = { [key: string]: string };
+
+export interface UseCustomizationPlaygroundProps {
+  customizationLabels: CustomizationOptions;
+  examples: {
+    [key: string]: {
+      [key: string]: string;
```

Comment:
Nitpick: IMHO, such a type doesn't have much benefit. Maybe we could add the available options? 🤔